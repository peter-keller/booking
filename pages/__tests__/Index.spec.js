import { shallowMount } from '@vue/test-utils'

import Homepage from '..'

describe('Page: Homepage', () => {
  const factory = (opts = {}) => shallowMount(Homepage, {
    ...opts
  })

  it('matches snapshot', () => {
    const wrapper = factory()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
