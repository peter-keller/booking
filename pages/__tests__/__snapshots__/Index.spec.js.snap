// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: Homepage matches snapshot 1`] = `"<h1>Homepage</h1>"`;
